recipe/meta.yaml

```diff
@@ -7,11 +7,10 @@ package:
 
 source:
   url: https://www.fossil-scm.org/xfer/uv/{{ name }}-src-{{ version }}.tar.gz
```

Comment:
upstream is now at 2.19. 
https://fossil-scm.org/home/uv/download.html

Reply:
They don't seem to have an easily accessible source anymore, much less branches, much less... 
All the more... bizarre... to [read](https://fossil-scm.org/home/doc/trunk/www/fossil-v-git.wiki) how fossil is better than git.

I think we might just have to bite the bullet and keep the hashes in
https://fossil-scm.org/home/tarball/1e131febd3fbb028d00cab6d020214e8fe36be95daaf93237523c29c542e9a5f/fossil-src-2.19.tar.gz

Reply:
> upstream is now at 2.19.

And now up to 2.20... i'm taking another go at it on #16, and found they _do_ have tags, so the URL can be very predictable (though not updateable)... i'll need to figure out a way to stay more on top of the versioning.

> bizarre

yep, it's an alternate universe, to be sure. but if it's good enough for shipping sqlite, it's good enough for _a lot_ of things.

but seriously, there's a lot to be said for "small" forges like `gitea` and `fossil`... the official distribution is a 3mb binary, and provides, in a skeletal way and for a pre-known set of repos, everything that e.g. github+gitter provides _except CI runners_... and can be run on a $5/mo VPS. Heck, a fossil peer will probably run fully in the browser, with some tweaks.